WooCommerce/Classes/POS/Presentation/PointOfSaleEntryPointView.swift

```diff
@@ -15,6 +15,7 @@ struct PointOfSaleEntryPointView: View {
     private let purchasableItemsSearchController: PointOfSaleSearchingItemsControllerProtocol
     private let couponsController: PointOfSaleCouponsControllerProtocol
     private let couponsSearchController: PointOfSaleSearchingItemsControllerProtocol
+    private let ordersController: PointOfSaleOrdersControllerProtocol
```

Comment:
Same here with `ordersController` vs `orderController`, maybe makes sense to rename the original one in the sense of order creation, fulfilment, factory, ... and this one as order list? Naming is indeed a difficult problem 😅

Reply:
Good points, I'll rename!